modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -258,6 +261,51 @@ export default {
     async create() {
       await this.edit(null);
     },
+    async handleUtilityOperation(action) {
+      const operation = this.utilityOperations.menu
+        .find((op) => op.action === action);
+
+      if (!operation) {
+        return;
+      }
+
+      const {
+        modal, body, ...modalOptions
+      } = operation.modalOptions || {};
+
+      if (modal) {
+        apos.modal.execute(modal, {
```

Comment:
Hmm, there's nothing that needs to happen after it, but awaiting it is best yes in case someone invoking this method (which claims to be async, after all) cares about timing.

Reply:
This should still be updated for clarity.